bin/hdfcoinc/pycbc_page_segplot

```diff
@@ -39,6 +39,8 @@ parser.add_argument('--segment-names', type=str, nargs="+", required=False,
                         help='Names of segments in the segment definer table.')
 parser.add_argument('--output-file', type=str,
                         help='Path of the output HTML file.')
+parser.add_argument('--ifos', nargs='+', default=['H1', 'L1'],
+                        help='List of IFOs: default H1, L1.')
```

Comment:
it looks like this must be entered a space-separated list, whereas the help suggests a comma-separated list. make this clear in help message as for page_segtable

Reply:
Done